Makefile

```diff
@@ -11,12 +11,20 @@ GORUN = env GO111MODULE=on go run
 libzkp:
 	cd $(PWD)/rollup/circuitcapacitychecker/libzkp && make libzkp
 
+libzstd:
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+	@sudo mv libzktrie.so /usr/local/lib
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
+	@sudo mv libscroll_zstd.so /usr/local/lib
+
 nccc_geth: ## geth without circuit capacity checker
```

Comment:
```suggestion
nccc_geth: libzstd ## geth without circuit capacity checker
```

Reply:
updated in [3640e89](https://github.com/scroll-tech/go-ethereum/pull/726/commits/3640e8938d2f03d23201fe9cf18bfe3adc41782f).